.github/workflows/ci.yml

```diff
@@ -122,7 +122,7 @@ jobs:
         key: "${{ env.ZC_TOOLCHAIN }}-${{ matrix.target }}-${{ env.FEATURES_HASH }}-${{ hashFiles('**/Cargo.lock') }}"
 
     - name: Check
-      run: cargo +${{ env.ZC_TOOLCHAIN }} check --manifest-path ${{ matrix.manifest-path }} --target ${{ matrix.target }} --features "${{ matrix.features }}" --verbose
+      run: cargo +${{ env.ZC_TOOLCHAIN }} check --manifest-path ${{ matrix.manifest-path }} --target ${{ matrix.target }} --features "${{ matrix.features }}" --verbose --all-targets
```

Comment:
Uber nit: Can you swap the flag order so it's `--all-targets --verbose` at the end?

Reply:
Actually, another thought: Can you also add `--all-targets` to the `cargo build` step?

Reply:
Might I suggest scrapping the `cargo check` step entirely in favor of `cargo build --all-targets`?

Reply:
Good question; I'll look into that and we can follow up in a separate PR if it turns out that's the way to go.